neomodel/exceptions.py

```diff
@@ -53,15 +56,21 @@ def __init__(self, db_node, current_node_class_registry):
         """
         self.db_node = db_node
         self.current_node_class_registry = current_node_class_registry
-        
+
     def __str__(self):
         node_labels = ",".join(self.db_node.labels)
-        ncr_items = list(map(lambda x:"{} --> {}".format(",".join(x[0]), x[1]),
-                             self.current_node_class_registry.items()))
+        ncr_items = list(
```

Comment:
This is just a style/preference thing, but it might be more readable to do the following:

```
ncr_items_multiline = "\n".join(
    "{} --> {}".format(",".join(node_labels), model_class)
    for node_labels, model_class in self.current_node_class_registry.items()
)
```